extension.ts

```diff
@@ -308,7 +308,7 @@ export async function activate(context: vscode.ExtensionContext) {
   // Initialize mode handler for current active Text Editor at startup.
   if (vscode.window.activeTextEditor) {
     let mh = await getAndUpdateModeHandler();
-    mh.updateView(mh.vimState, { drawSelection: false, revealRange: false });
+    await mh.updateView(mh.vimState, { drawSelection: false, revealRange: false });
```

Comment:
Don't need to await this as the stuff thereafter is not reliant on an updated view.